manager_synchronized.go

```diff
@@ -43,8 +43,6 @@ func (m *synchronizedProcessManager) ID() string {
 }
 
 func (m *synchronizedProcessManager) CreateProcess(ctx context.Context, opts *options.Create) (Process, error) {
-	opts.Synchronized = true
```

Comment:
why not leave this here, and then unwrap all of the returned processes? 

I think it's a bit undesireable to return the same process in multiple Get() calls, but have them be protected by different locks.

Reply:
Because that causes the original issue with being unable to call multiple methods in different goroutines on a single process. Requesting different copies of the same process (i.e. what's done in all the remote services) has to produce different synchronized processes.

Reply:
This feels unexpected. Let's take the modification of synchronized.True out